### PR TITLE
Updating README, fixing account_id bug

### PR DESCRIPTION
**Pull request by grahamjenson**
Fixing issue with assume-id that would not allow for inputting the actual id, also adding a test for it.

Updating Readme.

### CODE REVIEW

File: test/assume-role.bats [hunk 1]
```diff
@@ -47,6 +47,12 @@ setup() {
 }
 
 teardown() {
+  # This will output the if the test fails
+  for i in ""${!lines[@]}""
```
Comment (sds) [inline test/assume-role.bats]: What are these double'd double quotes doing?

**Review by sds — APPROVED** [review]
Looks fine but curious about one line that seems a bit odd?

File: test/assume-role.bats (hunk 1 above)
Comment (grahamjenson) [reply test/assume-role.bats]: should be single quotes

Comment (grahamjenson) [reply test/assume-role.bats]: fixed

**Pull request merged**